src/hooks/useCart/useCart.tsx

```diff
@@ -215,19 +219,17 @@ export const useCart = (
             identifiers.some(identifier => !identifier.value)
           ) {
             numUnverifiedTransactions += 1;
+          } else {
+            identiferValues.push(
+              ...identifiers.map(idenfitier => idenfitier.value)
```

Comment:
minor change! identifier spelling :)

Reply:
HAHAHA wah good catch, I keep spelling it wrong omg